server/test/routes/index.test.ts

```diff
@@ -75,3 +75,91 @@ describe('Test GET /v1/user/:userId/courses', () => {
     expect(courses.previous[0].courseCode).toBe('TU-A1100');
   });
 });
+
+describe('Test POST /v1/courses/:courseId/instances', () => {
```

Comment:
should we also test the case if responsible teacher does not exist?

Reply:
True! I added a test for that.